scr/main/java/bittercode/model/Cart.java

```diff
@@ -0,0 +1,31 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+
+public class Cart implements Serializable {
+
+    private Book book;
+    private int quantity;
+
+    private Cart(Book book, quantity) {
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Missing type declaration in constructor parameter</b></div>
<div id="fix">

The constructor parameter `quantity` is missing its type declaration. Consider adding `int` type to the parameter.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -10,4 +10,4 @@
 -    private Cart(Book book, quantity) {
 +    private Cart(Book book, int quantity) {
          this.book = book;
          this.quantity = quantity;
      }
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/1#issuecomment-2517477681>#c23dbc</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged